apps/vf-mvp/next.config.js

```diff
@@ -1,9 +1,49 @@
 const path = require('path');
 const DuplicatePackageCheckerPlugin = require('duplicate-package-checker-webpack-plugin');
+const nextSafe = require('next-safe');
 const withBundleAnalyzer = require('@next/bundle-analyzer')({
   enabled: process.env.ANALYZE === 'true',
 });
 
+const isDev = process.env.NODE_ENV !== 'production';
+const AUTH_GW_BASE_URL = isDev
+  ? 'https://virtualfinland-authgw.localhost'
+  : process.env.NEXT_PUBLIC_AUTH_GW_BASE_URL || '';
+const CODESETS_BASE_URL = isDev
+  ? 'http://localhost:3166'
+  : process.env.NEXT_PUBLIC_CODESETS_BASE_URL || '';
+
+// https://trezy.gitbook.io/next-safe/usage/configuration
+const nextSafeConfig = {
+  isDev,
+  contentTypeOptions: 'nosniff',
+  contentSecurityPolicy: {
+    'base-uri': "'none'",
+    'child-src': "'none'",
+    'connect-src': ["'self'", AUTH_GW_BASE_URL, CODESETS_BASE_URL],
+    'default-src': "'self'",
+    'font-src': ["'self'", 'https://fonts.gstatic.com/'],
+    'form-action': "'self'",
+    'frame-ancestors': "'none'",
+    'frame-src': "'none'",
+    'img-src': "'self'",
+    'manifest-src': "'self'",
+    'media-src': "'self'",
+    'object-src': "'none'",
+    'prefetch-src': "'self'",
```

Comment:
Tämän vois tiputella pois vanhentuneena: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/prefetch-src

> Disable the prefetch-src directive
> "prefetch-src": false,
https://trezy.gitbook.io/next-safe/configuration-options/contentsecuritypolicy#disable-the-prefetch-src-directive